### PR TITLE
Quick buttons borders fix & more

### PR DESCRIPTION
**Pull request by AlexAzumi**
Quick buttons borders were changed to fit inside the `dialogbox` and `poems`.

The chapter 30 had a syntaxis error in Monika's replicas, on the other hand, some changes were not merged because of me.

> I'm so sorry :(

### CODE REVIEW

File: game/gui.rpy [hunk 1]
```diff
@@ -193,7 +193,7 @@ define -2 gui.confirm_button_text_xalign = 0.5
 define -2 gui.page_button_borders = Borders(10, 4, 10, 4)
 
 
-define -2 gui.quick_button_text_size = 16
+define -2 gui.quick_button_text_size = 19
```
Comment (saber-nyan) [inline game/gui.rpy:196]: I'm not sure that the text of the Russian translation will fit in now...

File: game/script-ch30.rpy [hunk 2]
```diff
@@ -312,7 +312,7 @@ label ch30_main2:
     m "It kind of freaked me out, how easy it was."
     m "Well, you're playing on Android, so it was actually a bit less difficult..."
     m "I had to go to 'Settings' and find the 'Characters' button..."  # TODOne: monika fix
-    elif renpy.macintosh:
+    if renpy.macintosh:
```
Comment (saber-nyan) [inline game/script-ch30.rpy:315]: Oh, I didn't notice.

**Review by saber-nyan — APPROVED** [review]
Many thanks!

File: game/gui.rpy (hunk 1 above)
Comment (AlexAzumi) [reply game/gui.rpy:196]: Ohhh, I didn't think about translations, but it might not be a big deal.

**Pull request merged**